src/universal-dapp.js

```diff
@@ -749,8 +750,9 @@ UniversalDApp.prototype.runTx = function (args, cb) {
       tx.sign(account.privateKey);
       var block = new EthJSBlock({
         header: {
-          // FIXME: support coinbase, difficulty, number and gasLimit
-          timestamp: new Date().getTime() / 1000 | 0
+          // FIXME: support coinbase, difficulty and gasLimit
+          timestamp: new Date().getTime() / 1000 | 0,
+          number: self.blockNumber++,
```

Comment:
I don't know about `runCall`, but I inspected a `args.useCall` that is true on constants and false on transactions, I guess that's that.